### PR TITLE
Update 01-welcome.md

### PR DESCRIPTION
**Pull request by raynamharris**
I added one objective and one key point to help elaborate on the goal and rational for this episode.

Okay @ChristinaLK ?

### CODE REVIEW

File: _episodes/01-welcome.md [hunk 1]
```diff
@@ -8,8 +8,10 @@ questions:
 - "What will we cover (or not) in this workshop?"
 - "What prior knowledge do workshop participants have?"
 objectives:
+- "Explain what made their favorite class(es) so great."
```
Comment (ChristinaLK) [inline _episodes/01-welcome.md]: Maybe: "Identify a component of successful teaching by reflecting on previous learning experiences."

**Comment by ChristinaLK** [comment]
+1 to key point.

File: _episodes/01-welcome.md (hunk 1 above)
Comment (lexnederbragt) [inline _episodes/01-welcome.md]: Or "Reflecting on previous learning experiences to identify components of successful teaching" ?

Comment (ChristinaLK) [inline _episodes/01-welcome.md]: That's not an objective, it's a method.  Maybe the objective should just be "identify a component of successful teaching."

Comment (lexnederbragt) [inline _episodes/01-welcome.md]: You're right...! 'a component' or 'components', though?

Comment (ChristinaLK) [inline _episodes/01-welcome.md]: one or more components.  ;)

Comment (lexnederbragt) [inline _episodes/01-welcome.md]: OK :-)

**Comment by ErinBecker** [comment]
Merged! Thanks @raynamharris.

**Pull request merged**